src/normalize/pathoverlay.ts

```diff
@@ -164,7 +183,8 @@ export class PathOverlayNormalizer implements NonFacetUnitNormalizer<UnitSpecWit
           ...pick(markDef, ['clip', 'tooltip']),
           ...pointOverlay
         },
-        encoding: overlayEncoding
+        // drop label from encoding when having detail (grouped line chart)
```

Comment:
I'm not sure if I understand this comment. 

What do you mean "when having detail (grouped line chart)"? 
Given the code below doesn't do any conditional check.